config/install/field.field.node.landing_page_lb.field_meta_image.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_meta_image
+    - media.type.image
+    - node.type.landing_page_lb
+id: node.landing_page_lb.field_meta_image
+field_name: field_meta_image
+entity_type: node
+bundle: landing_page_lb
+label: 'Meta image'
+description: 'An image associated with this page, for use as a thumbnail in social networks and other services. Images should be at least 1080px in width.'
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:media'
+  handler_settings:
+    target_bundles:
+      image: image
+    sort:
+      field: changed
+      direction: DESC
+    auto_create: true
+    auto_create_bundle: ''
+field_type: entity_reference
```

Comment:
suggestion(optional): we can break dependency on config level and make code by leveraging openy_er module functionality @froboy

Reply:
@podarok from what I can tell, `openy_er` works on Content references and this is a Media reference so I think that is unnecessary?

Reply:
we can have openy_er for other entities as well. But this is minor in current case